tests/sass/function-name-format.scss

```diff
@@ -0,0 +1,12 @@
+$foo: kabab-case();
+$foo: snake_case();
+$foo: camelCase();
+$foo: PascalCase();
+$foo: Camel_Snake_Case();
+$foo: SCREAMING_SNAKE_CASE();
+$foo: _with-leading-underscore();
+$foo: _does_NOT-fitSTANDARD();
+
```

Comment:
can you find the Sass equivalent for your example? I can't figure out the right syntax.